docs/tutorial/quick-start.rst

```diff
@@ -23,35 +25,16 @@ more in-depth installation instructions, refer to :ref:`rocm-install-overview`.
                 {% for (os_version, os_release) in config.html_context['ubuntu_version_numbers'] %}
                 .. tab-item:: {{ os_version }}
 
-                    .. tab-set::
-
-                        .. tab-item:: Native package manager
-                            :sync: native-package-manager
-
-                            .. code-block:: bash
-                                :substitutions:
-
-                                sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
-                                sudo usermod -a -G render,video $LOGNAME # Adding current user to Video, Render groups. See prerequisites.
-                                wget https://repo.radeon.com/amdgpu-install/|amdgpu_version|/ubuntu/{{ os_release }}/amdgpu-install_|amdgpu_install_version|_all.deb
-                                sudo apt install ./amdgpu-install_|amdgpu_install_version|_all.deb
-                                sudo apt update
-                                sudo apt install amdgpu-dkms
-                                sudo apt install rocm
-                                echo "Please reboot system for all settings to take effect."
+                    .. code-block:: bash
+                        :substitutions:
 
-                        .. tab-item:: AMDGPU installer
-                            :sync: amdgpu-installer
-
-                            .. code-block:: bash
-                                :substitutions:
-
-                                sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
-                                sudo usermod -a -G render,video $LOGNAME # Adding current user to Video, Render groups. See prerequisites.
-                                sudo apt update
-                                wget https://repo.radeon.com/amdgpu-install/|amdgpu_version|/ubuntu/{{ os_release }}/amdgpu-install_|amdgpu_install_version|_all.deb
-                                sudo apt install ./amdgpu-install_|amdgpu_install_version|_all.deb
-                                sudo amdgpu-install --usecase=graphics,rocm
+                        sudo apt update
+                        sudo apt install "linux-headers-$(uname -r)" "linux-modules-extra-$(uname -r)"
+                        # Add the current user to the render and video groups (see prerequisites).
```

Comment:
I initially had this comment on a separate line.  Feedback was to put it on the same line.  Let's see if others give the same feedback again :)

Reply:
Do we really need to have the comment at all?

Reply:
we added it from Saad's feedback, if you recall.

Reply:
Right.  I forgot where that came from.  Then maybe we should leave it as a single line.